### PR TITLE
Updated the k8s-ei-operator documentation

### PR DESCRIPTION
**Pull request by sajithaliyanage**

### CODE REVIEW

File: en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md [hunk 1]
```diff
@@ -7,48 +7,333 @@ The k8s-ei-operator allows you to deploy your integration solutions as an **Inte
...
 The k8s-ei-operator is built with operator-sdk v0.7.0 and supported in the following environment.
 
->   [Kubernetes](https://kubernetes.io/docs/setup/) cluster and client v1.11+
+-   [Kubernetes](https://kubernetes.io/docs/setup/) cluster and client v1.11+   
+-   [Docker](https://docs.docker.com/)
 
-## Install k8s-ei-operator
-Please do the following steps (one time) to configure the Kubernetes environment in your server/machine.
+## Install EI Operator
+Please do the following steps to intall the EI Kubernetes operator in your Kubernetes environment.
```
Comment (menuka94) [inline en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md]:
```suggestion
Please do the following steps to install the EI Kubernetes operator in your Kubernetes environment.
```

File: en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md [hunk 2]
```diff
@@ -7,48 +7,333 @@ The k8s-ei-operator allows you to deploy your integration solutions as an **Inte
...
+
+```bash
+$ kubectl get deployment
+
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+k8s-ei-operator     1/1     1            1          1m
+```
+
+## Deploy and Run Integration Solutions with EI Operator
+EI operator supports to deploy and run your integration solutions in Kubernetes environemnt.
```
Comment (menuka94) [inline en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md]:
```suggestion
EI operator supports deployment and running your integration solutions in a Kubernetes environment. 
```

File: en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md [hunk 3]
```diff
@@ -7,48 +7,333 @@ The k8s-ei-operator allows you to deploy your integration solutions as an **Inte
...
+$ kubectl get deployment
+
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+k8s-ei-operator     1/1     1            1          1m
+```
+
+## Deploy and Run Integration Solutions with EI Operator
+EI operator supports to deploy and run your integration solutions in Kubernetes environemnt. 
+
+Here we have demonstrate an example of **HelloWorld** service which response as `{"Hello":"World"}` to the user request. This scenario can be created using `integration_cr.yaml` format as given below.
```
Comment (menuka94) [inline en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md]:
```suggestion
Here we have demonstrated an example of **HelloWorld** service which responds `{"Hello": "World"}` to the user request. This scenario can be created using `integration_cr.yaml` format as given below.
```

Comment (sajithaliyanage) [reply en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md]: Done

File: en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md (hunk 2 above)
Comment (sajithaliyanage) [reply en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md]: Done

File: en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md (hunk 1 above)
Comment (sajithaliyanage) [reply en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md]: Done

**Pull request merged**